frame/support/test/src/pallet_version.rs

```diff
@@ -1,29 +0,0 @@
-// This file is part of Substrate.
-
-// Copyright (C) 2020-2022 Parity Technologies (UK) Ltd.
-// SPDX-License-Identifier: Apache-2.0
-
-// Licensed under the Apache License, Version 2.0 (the "License");
-// you may not use this file except in compliance with the License.
-// You may obtain a copy of the License at
-//
-// 	http://www.apache.org/licenses/LICENSE-2.0
-//
-// Unless required by applicable law or agreed to in writing, software
-// distributed under the License is distributed on an "AS IS" BASIS,
-// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-// See the License for the specific language governing permissions and
-// limitations under the License.
-
-use frame_support::{crate_to_pallet_version, traits::PalletVersion};
-
-#[test]
-fn ensure_that_current_pallet_version_is_correct() {
```

Comment:
Why is this test removed?

Reply:
This test is not included in ·frame-support-test·, I guess someone forgot to delete it.

Reply:
Why would your first instinct be to remove the test? Doesn't it make more sense to include this test instead? Or did you think this test is useless?

Reply:
the `crate_to_pallet_version ` method has been removed